app/assets/stylesheets/cclow/_geography-page.scss

```diff
@@ -44,12 +45,12 @@
         margin-right: 1.5rem;
       }
     }
+  }
 
-    .highlight {
-      color: $red;
-      font-family: "BrownTT Bold";
-      margin: 0 4px;
-    }
+  .highlight {
+    color: $red;
+    font-family: "BrownTT Bold";
```

Comment:
You can use the variable `$family-sans-serif-bold`.

Reply:
thanks)
I fixed it